components/atoms/Select/single-select.tsx

```diff
@@ -64,7 +64,7 @@ const SingleSelect = ({
         )}
       >
         <Tooltip content={current?.label ?? placeholder}>
-          <div className="flex items-center w-fit max-w-48">
+          <div className="flex items-center  w-44 ">
```

Comment:
nit: Extra space here

```suggestion
          <div className="flex items-center w-44 ">
```